data/notices.json

```
@@ -727,6 +727,18 @@
         }
       ],
       "schemaVersion": "1"
+    },
+    {
+      "title": "(credentials): CLI fails to validate credentials under certain conditions",
```

```suggestion
      "title": "(cli): may fail on credentials coming from credential provider plugins",
```

Please also change the title of the https://github.com/aws/aws-cdk/issues/32653 to start with `(cli)` (`credentials` is not a component...)